datasets/narrativeqa_manual/README.md

```diff
@@ -0,0 +1,194 @@
+
+- **Homepage:** [NarrativeQA Homepage](https://deepmind.com/research/open-source/narrativeqa)
+- **Repository:** [NarrativeQA Repo](https://github.com/deepmind/narrativeqa)
+- **Paper:** [The NarrativeQA Reading Comprehension Challenge](https://arxiv.org/pdf/1712.07040.pdf)
+- **Leaderboard:**
+- **Point of Contact:** [Tomáš Kočiský](mailto:tkocisky@google.com) [Jonathan Schwarz](mailto:schwarzjn@google.com) [Phil Blunsom](pblunsom@google.com) [Chris Dyer](cdyer@google.com) [Karl Moritz Hermann](mailto:kmh@google.com) [Gábor Melis](mailto:melisgl@google.com) [Edward Grefenstette](mailto:etg@google.com)
+
+### Dataset Summary
+
+NarrativeQA Manual is an English-language dataset of stories and corresponding questions designed to test reading comprehension, especially on long documents. THIS DATASET REQUIRES A MANUALLY DOWNLOADED FILE! Because of a script in the original repository which downloads the stories from original URLs everytime, the links are sometimes broken or invalid. Therefore, you need to manually download the stories for this dataset using the script provided by the authors (https://github.com/deepmind/narrativeqa/blob/master/download_stories.sh). Running the shell script creates a folder named "tmp" in the root directory and downloads the stories there. This folder containing the stories can be used to load the dataset via `datasets.load_dataset("narrativeqa_manual", data_dir="<path/to/folder>")`.
+
+### Supported Tasks and Leaderboards
+
+The dataset is used to test reading comprehension. There are 2 tasks proposed in the paper: "summaries only" and "stories only", depending on whether the human-generated summary or the full story text is used to answer the question.
+
+### Languages
+
+English
+
+## Dataset Structure
+
+### Data Instances
+
+A typical data point consists of a question and answer pair along with a summary/story which can be used to answer the question. Additional information such as the url, word count, wikipedia page, are also provided.
+
+A typical example looks like this:
+```
+{
+    "document": {
+        "id": "23jncj2n3534563110",
+        "kind": "movie",
+        "url": "https://www.imsdb.com/Movie%20Scripts/Name%20of%20Movie.html",
+        "file_size": 80473,
+        "word_count": 41000,
+        "start": "MOVIE screenplay by",
+        "end": ". THE END",
+        "summary": {
+            "text": "Joe Bloggs begins his journey exploring...",
+            "tokens": ["Joe", "Bloggs", "begins", "his", "journey", "exploring",...],
+            "url": "http://en.wikipedia.org/wiki/Name_of_Movie",
+            "title": "Name of Movie (film)"
+        },
+        "text": "MOVIE screenplay by John Doe\nSCENE 1..."
+    },
+    "question": {
+        "text": "Where does Joe Bloggs live?",
+        "tokens": ["Where", "does", "Joe", "Bloggs", "live", "?"],
+    },
+    "answers": [
+        {"text": "At home", "tokens": ["At", "home"]},
+        {"text": "His house", "tokens": ["His", "house"]}
+    ]
+}
```

Comment:
Here we expect an actual example from the dataset object, but the format of this example is not the one used in the `narrativeqa_manual` script at the moment.

However if we switch to using the same `features` as in the `narrativeqa` script then this would be a valid example. And on the contrary if we stick with the current features of `narrativeqa_manual`, we would need to replace this example by a valid one.